src/providers/inventory-data.ts

```diff
@@ -41,19 +41,23 @@ export class InventoryData {
   }
 
   getBrands() {
```

Comment:
This file seems repetitive. The only difference between each function is which link it uses, so could you abstract it behind a function that receives the link as a parameter?